src/components/Editor/CodeEditor.tsx

```diff
@@ -10,7 +10,7 @@ import randomColor from 'randomcolor';
 import ClientCursor from './ClientCursor';
 
 import { IAppState } from '../../store/store';
-import { AttachDocAction, loadDocAction } from '../../actions/docActions';
+import { AttachDoc, loadDocAction } from '../../actions/docActions';
```

Comment:
Is there any reason to change the name of AttachDocAction? If there is any reason, why not change the names of other actions at once?

Reply:
the master branch has the same name called AttachDocAction, so Lint requested a change.

https://github.com/yorkie-team/yorkie-codepair/blob/master/src/actions/docActions.ts#L13
https://github.com/yorkie-team/yorkie-codepair/blob/master/src/actions/docActions.ts#L31

If all actions should have an action value in the suffix, I will change the [13th line](
https://github.com/yorkie-team/yorkie-codepair/blob/master/src/actions/docActions.ts#L13)


Currently, I had changed the same as below
https://github.com/yorkie-team/yorkie-codepair/blob/master/src/actions/peerActions.ts#L28

Reply:
Thanks for explaining why.

For consistency of variable names, please change them for this time.
- Rename `AttachDoc` -> `attachDocAction`
- Rename `AttachDocAction` -> `IAttachDocAction`

I think it would be nice to find an appropriate style guide later and change the style of all names at once to change the naming convention.